sql/moz-fx-data-shared-prod/udf/ga_is_mozilla_browser/udf.sql

```diff
@@ -0,0 +1,23 @@
+/*
+
+Determine if a browser in a Google Analytics data is one produced by Mozilla
+
+*/
+CREATE OR REPLACE FUNCTION udf.ga_is_mozilla_browser(browser STRING)
+RETURNS BOOLEAN AS (
+  CASE
+  WHEN
+    browser LIKE 'Firefox'
```

Comment:
Is using `LIKE` instead of `=` a conscious decision?  I don't think there's any functional difference since there's no pattern matching but I think `LIKE` is slightly less performant.

Reply:
No, just an error on my part. I'll change it to `=` to keep consistency with the previous behaviour